Additional factors for facilitating NFT market verification

```diff
@@ -0,0 +1,80 @@
+|CIP|Title|Authors|Comments-Summary|Comments-URI|Status|Type|Created|License|Requires|
```

Comment:
There is another, simpler header format which doesn't rely on fields matching in a table.  I hope @KtorZ @SebastienGllmt will correct me if I'm wrong but I think the parsing of CIP information depends on a header format like this (seen at the beginning of the raw Markdown for each of our merged CIP's):

```
---
CIP: 1
Title: CIP process
Authors: ...
```